flask/test/endpoint_tests/test_box.py

```diff
@@ -1,73 +1,32 @@
 import pytest
 
 
-@pytest.mark.usefixtures("default_qr_code")
 @pytest.mark.usefixtures("default_box")
-def test_get_box_from_code(client, default_box, default_qr_code):
-    code = default_qr_code["code"]
-    graph_ql_query_string = f"""query Box {{
-                getBoxDetails(qrCode: "{code}") {{
+def test_get_box_from_id(client, default_box):
```

Comment:
test_get_box_from_id seems like a confusing name. Usually id refers to the internal id in the DB, but in this case it looks like you mean the boxLabelIdentifier, which essentially means the number written on the outside of the box. It would better if the test clarified that, or had a comment explaining what it did, or both.